templates/package-json.js

```diff
@@ -1,16 +1,69 @@
 module.exports = (answers) => {
-	return `{
-	"private": true,
-	"name": "${answers.name}",
-	"devDependencies": {
-		"eslint": "^7.1.0",
-		"eslint-config-origami-component": "^2.0.1",
-		"origami-ci-tools": "^2.0.0",
-		"remark": "^12.0.0",
-		"remark-lint": "^7.0.0",
-		"remark-preset-lint-origami-component": "^1.1.1",
-		"stylelint": "^13.5.0",
-		"stylelint-config-origami-component": "^1.0.2"
+
+	const packageJson = {
+		"name": `@financial-times/${answers.name}`,
+		"version": "0.0.0",
+		"description": answers.description,
+		"keywords": JSON.stringify(answers.keywords),
+		"homepage": `https://registry.origami.ft.com/components/${answers.name}`,
+		"bugs": {
+			"url": `https://github.com/Financial-Times/${answers.name}/issues`
+		},
+		"repository": {
+			"type": "git",
+			"url": `git+https://github.com/Financial-Times/${answers.name}.git`
+		},
+		"license": "MIT",
+		"author": "",
+		"type": "module",
+		"devDependencies": {
+			"remark": "^12.0.1",
+			"remark-lint": "^7.0.1",
+			"remark-preset-lint-origami-component": "^2.0.0-beta.1"
+		},
+		"dependencies": {},
+		"peerDependencies": {},
+		"scripts": {},
+		"engines": {
+			"npm": ">=7"
+		}
+	};
+
+	if (answers.javascript) {
+		Object.assign(packageJson, {
+			"exports": {
+				".": {
+					"browser": "./main.js"
+				}
+			},
+			"browser": "./main.js",
+			"types": "./types/main.d.ts"
+		});
```

Comment:
Let's remove the typescript parts for now, we haven't added it to any of the spec v2 components we are currently working on
```suggestion
		Object.assign(packageJson, {
			"browser": "./main.js",
		});
```